src/style.css

```diff
@@ -1 +1,96 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, sans-serif;
```

Comment:
The `font-family` property should not include multiple font names within the `@font-face` rule. Instead, specify only the custom font name, such as 'Avenir'.